playbooks/06-deploy-edpm.yml

```diff
@@ -6,6 +6,11 @@
     step: pre_deploy
   ansible.builtin.import_playbook: ./hooks.yml
 
+- name: Deploy NFS server on target nodes
+  when:
```

Comment:
you want to move that `when` in a pre_tasks of the imported playbook, else we'll end with a list of "skipping" tasks.

Reply:
I was waiting for your PR to use the same pattern here :)